src/App.tsx

```diff
@@ -9,7 +11,9 @@ const App: React.FC = () => (
     <Routes>
       <Route path='/' element={<Landing />} />
       <Route path='*' element={<ErrorView />} />
+      <Route path='./team' element={<Team />} />
```

Comment:
Change this to `/team` or just `team`. Also this doesn't matter a whole lot, but I'd prefer leaving the last path as the 404 error view